backend/src/packages/chaiNNer_pytorch/settings.py

```diff
@@ -66,12 +66,24 @@
     ),
 )
 
+package.add_setting(
+    NumberSetting(
+        label="Memory Budget Limit (GiB)",
+        key="budget_limit",
+        description="Maximum memory (VRAM) to use for PyTorch inference. 0 means no limit. Memory usage measurement is not completely accurate yet; you may need to significantly adjust this budget limit via trial-and-error if it's not having the effect you want.",
```

Comment:
1. This limits RAM when using the CPU.
2. This currently only affects the Auto tile size mode.